src/test/rustdoc/async-fn.rs

```diff
@@ -52,3 +53,31 @@ pub trait Trait<const N: usize> {}
 // @has async_fn/fn.const_generics.html
 // @has - '//pre[@class="rust fn"]' 'pub async fn const_generics<const N: usize>(_: impl Trait<N>)'
 pub async fn const_generics<const N: usize>(_: impl Trait<N>) {}
+
+// test that elided lifetimes are properly elided and not displayed as `'_`
```

Comment:
Can you add some tests with named lifetimes to show that we don't throw them out?

(It's pretty clear from the implementation that we won't, but this is future proofing!)

Reply:
Done, good idea.